### DISCUSSION TITLE
Compte rendus meeting

### DISCUSSION THREAD

**Discussion by neonima** [devcherchecollegue-org/rust-pong#3] [General]
_ici seront les comptes rendus de nos sessions vocales_

**Comment by Fred-si** [comment]
Je vous mes là les notes que j’ai prises sur ce dont on a discuté mercredi soir (3 mars 2021)

 
- Architecture client-server
- Réconciliation : le serveur a le dernier mot
- Temps réel simulé par échantillonnage
- Communication via websockets
- Front
  + Envoie position + vitesse raquette au serveur
  + Calcule le mouvement de la balle
  
- Back
  + La position de la balle est recalculé périodiquement (tick) et renvoyé aux
    clients.
  + On crée une parti quand un client se connecte et on attend un deuxième
    joueur, la parti démarre quand 2 joueurs sont connectés.
  
- On écrit des tests ou pas ?
  
- Les pulls requests en français sont autorisés

- Point étape mercredi 21 h si bon pour tout le monde
  + Définir des objectifs pour les points étapes
- un participant pense que ça pourrait être bien de prévoir mise en place de
  release notes.
